.github/workflows/mlmodeldownloader.yml

```diff
@@ -81,7 +81,7 @@ jobs:
         flags: [
           '--use-modular-headers',
           # Tests are skipped since the Swift tests need modules.
-          '--skip-tests --use-libraries'
+          '--skip-tests'
```

Comment:
Just delete this line. No reason to repeat without any tests.

Reply:
Good call, I was able to simplify it further since there was only one flag left and just added it directly in the command.